src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -765,7 +765,7 @@ internal void UpdateMembraneButtons(string membrane)
 
     internal void UpdateRigiditySlider(float value, int mutationPoints)
     {
-        if (mutationPoints > 0)
+        if (mutationPoints >= 5)
```

Comment:
Please make the amount of MP it costs per step a value in Constants.
Also I think the cost should be something like 2 MP per step so that it is easier to consume all the remaining MP.